Update contributing-to-a-project.md (#32781)
github/docs: content/get-started/exploring-projects-on-github/contributing-to-a-project.md

This tutorial uses [the Spoon-Knife project](https://github.com/octocat/Spoon-Knife), a test repository that's hosted on {% data variables.product.prodname_dotcom_the_website %} that lets you test the fork and pull request workflow.

> ### Creating a new fork
> 

Reviewer: ```suggestion
```

1. Navigate to the `Spoon-Knife` project at https://github.com/octocat/Spoon-Knife.
1. In the top-right corner of the page, click **Fork**.

   ![Screenshot of the main page of repository. A button, labeled with a fork icon and "Fork 59.3k," is outlined in dark orange.](/assets/images/help/repository/fork-button.png)

> 1. Under "Owner," select the dropdown menu and click an owner for the forked repository. (If your name is greyed out, it's because you have forked this project in the past; see "Updating an existing fork" below.)

Reviewer: ```suggestion
1. Under "Owner," select the dropdown menu and click an owner for the forked repository.
   >[!NOTE] If your username is grayed out, it's because the fork already exists. Instead, you should bring your existing fork up to date. For more information, see "[AUTOTITLE](/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork#syncing-a-fork-branch-from-the-web-ui)".
```

Reviewer: This seems like valuable guidance for beginners 💛 Let's put it in a note form, and link out to the supporting article ✨

Author: I agree, this seems very sensible. Let's do it the way you suggest.

1. In the top-right corner of the page, click **Fork**.

   ![Screenshot of the main page of repository. A button, labeled with a fork icon and "Fork 59.3k," is outlined in dark orange.](/assets/images/help/repository/fork-button.png)

> 1. Under "Owner," select the dropdown menu and click an owner for the forked repository. (If your name is greyed out, it's because you have forked this project in the past; see "Updating an existing fork" below.)

Reviewer: ```suggestion
1. Under "Owner," select the dropdown menu and click an owner for the forked repository.
   >[!NOTE] If your username is grayed out, it's because the fork already exists. Instead, you should bring your existing fork up to date. For more information, see "[AUTOTITLE](/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork#syncing-a-fork-branch-from-the-web-ui)".
```

{% endnote %}

> ### Updating an existing fork
> 
> If you forked the Spoon-Knife repository in the past, you can't create another fork from the same username. Instead you need to bring your fork up to date:
> 1. In the top right corner of the page, click on the arrow next to the **Fork** button.
> 1. A drop-down menu **Existing forks** appears. Click on your existing fork.
> 1. Click on the **Sync fork** button.
> ![Screenshot of the main page of repository. The "Existing fork" and "Sync fork" buttons are outlined in dark orange.](https://github.com/github/docs/assets/52163138/7180c512-f520-4d00-b847-4713a8b5b863)

Reviewer: ```suggestion
```

{% endnote %}

> 

Reviewer: ```suggestion
```

{% endnote %}

> ### Updating an existing fork
> 
> If you forked the Spoon-Knife repository in the past, you can't create another fork from the same username. Instead you need to bring your fork up to date:
> 1. In the top right corner of the page, click on the arrow next to the **Fork** button.
> 1. A drop-down menu **Existing forks** appears. Click on your existing fork.
> 1. Click on the **Sync fork** button.
> ![Screenshot of the main page of repository. The "Existing fork" and "Sync fork" buttons are outlined in dark orange.](https://github.com/github/docs/assets/52163138/7180c512-f520-4d00-b847-4713a8b5b863)
> 

Reviewer: ```suggestion
```

Reviewer: If we link out to "Syncing a fork" from the note in the procedure, we can remove this sub-section and ensure this article remains focused on helping users get started with forks for the first time :yellow_heart:
